benchmarks/TFLite/linux-x86_64.cmake

```diff
@@ -304,3 +304,145 @@ iree_benchmark_suite(
   RUNTIME_FLAGS
     "--task_topology_group_count=8"
 )
+
+# CPU, LLVM, local-sync, x86_64, full-inference
+iree_benchmark_suite(
+  GROUP_NAME
+    "linux-x86_64"
+
+  MODULES
+    "${DEEPLABV3_FP32_MODULE}"
+    "${MOBILESSD_FP32_MODULE}"
+    "${POSENET_FP32_MODULE}"
+    "${MOBILEBERT_FP32_MODULE}"
+    "${MOBILENET_V2_MODULE}"
+    "${MOBILENET_V3SMALL_MODULE}"
+    "${MOBILEBERT_INT8_MODULE}"
+    "${PERSON_DETECT_INT8_MODULE}"
+    "${EFFICIENTNET_INT8_MODULE}"
+
+  BENCHMARK_MODES
+    "full-inference,experimental-flags,aggressive-fusion"
```

Comment:
So originally when we made the experimental/default split the idea was that we would have at most one experimental configuration at a time. This was partially to do with the use of natural keys in benchmarks, but also to keep benchmarking time in check. See the comment in https://github.com/iree-org/iree/pull/7580/files#diff-f422d90311715986285ad57cdb9c40c9e9309d20a283de6762404da0dda2ab58R231-R232, for instance

Reply:
Yeah, can we add the flag(s) to the existing experimental configuration instead? I'm not sure how useful it would be to add longitudinal tracking for specific flags like this.